### ISSUE TITLE
Cannot install with ember-cli 0.2.0-beta.1

### ISSUE THREAD

**Issue by ninjatronic** [joliss/broccoli-sass#50]
Hi,

Not sure if this should be here or in ember-cli - if this is not the place I'll move the report over there.

This is an install into a freshly generated app using `ember-cli@0.2.0-beta.1`...

```
$ ember install:npm broccoli-sass
version: 0.2.0-beta.1
Installing packages for tooling via npm..Binary downloaded and installed at /Users/pete/ninjatronic/spikes/node_modules/broccoli-sass/node_modules/node-sass/vendor/darwin-x64/binding.node
Installing packages for tooling via npm...`darwin-x64` exists; testing
module.js:355
  Module._extensions[extension](this, filename);
                               ^
Error: Module did not self-register.
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/pete/ninjatronic/spikes/node_modules/broccoli-sass/node_modules/node-sass/lib/index.js:181:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
node-sass@1.2.3 postinstall: `node scripts/build.js`
Exit status 1
Error: node-sass@1.2.3 postinstall: `node scripts/build.js`
Exit status 1
    at EventEmitter.<anonymous> (/Users/pete/ninjatronic/spikes/node_modules/ember-cli/node_modules/npm/lib/utils/lifecycle.js:212:16)
    at EventEmitter.emit (events.js:110:17)
    at ChildProcess.<anonymous> (/Users/pete/ninjatronic/spikes/node_modules/ember-cli/node_modules/npm/lib/utils/spawn.js:14:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1008:16)
    at Process.ChildProcess._handle.onexit (child_process.js:1080:5)
```

**Comment by anler** [comment]
@ninjatronic I was using node v0.12.0 and in order to fix it I had to switch back to v0.10.36

**Comment by ninjatronic** [comment]
@anler thanks for the workaround - I too am using node v0.12...

```
$ node -v
v0.12.0
```

**Comment by ventismith** [comment]
@anler @ninjatronic You could also find someone who's forked the project and fixed it on their end with a pull request, such as:
````
npm install hipertracker/broccoli-sass --save-dev
````
Still a bit of a pain for projects that have it as a dependency though.

**Comment by anler** [comment]
@ventismith Good to know, thanks!

**Comment by ninjatronic** [comment]
@ventismith thanks, referencing the PR here so people can see that the work is coming :)

https://github.com/joliss/broccoli-sass/pull/49

**Comment by denniszhao** [comment]
Any hope for switching to node-sass v2.0.0+ (supports v0.12)?

**Comment by denniszhao** [comment]
Sorry, just read this https://github.com/joliss/broccoli-sass/pull/43

**Comment by aexmachina** [comment]
This issue is resolved by aexmachina/ember-cli-sass v3.1.0. See #47 for more details.
